src/structures/GuildMember.js

```diff
@@ -83,6 +89,7 @@ class GuildMember extends Base {
     if ('joined_at' in data) this.joinedTimestamp = new Date(data.joined_at).getTime();
     if ('premium_since' in data) this.premiumSinceTimestamp = new Date(data.premium_since).getTime();
     if ('roles' in data) this._roles = data.roles;
+    if ('pending' in data) this.pending = data.pending;
```

Comment:
Can't we just do `this.pending = data.pending ?? false`? cc @kyranet

Reply:
We can, as a matter of a fact, #5132 makes all the lines above use `??`.

Reply:
Done